packages/upgrade/test/__fixtures__/hooks/with-invalidate.ts

```
@@ -0,0 +1,9 @@
+import { trpc } from './trpc';
```

ESLint

[error] Filename is not in camel case. Rename it to `withInvalidate.ts`.

(unicorn/filename-case)